app/test_main.py

```diff
@@ -1,3 +1,21 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected_result", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (100, 100, [21, 17])
```

Comment:
Consider adding test cases for edge situations, such as negative numbers or very large numbers, to ensure the function handles data out of the normal range correctly, as suggested in the checklist.